apps/communications/dialer/test/marionette/lib/views/keypad/views.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+/* global module */
+var KeypadAccessor = require('./accessors');
+
+function KeypadView(client) {
+  
+  this.client = client;
+  this.accessors = new KeypadAccessor(client);
+
+}
+
+KeypadView.prototype = {
+ 
+  key: function(keyNumber) {
+    return this.client.helper.waitForElement
+      (this._getKeypadKeySelector(keyNumber));
+  },
+  _getKeypadKeySelector: function(key) {
```

Comment:
My bad, my first comment wasn't clear. This function exposes a selector. I think it has to be among the other selectors, in the accessors.